packages/react-checkbox/etc/react-checkbox.api.md

```diff
@@ -19,22 +19,21 @@ export const Checkbox: ForwardRefComponent<CheckboxProps>;
 export const checkboxClassName = "fui-Checkbox";
 
 // @public (undocumented)
-export interface CheckboxCommons {
-    checked: 'mixed' | boolean;
+export type CheckboxCommons = {
```

Comment:
Just curious, why the change from `interface` to `type`?

Reply:
We've mostly standardized on `type` in our new projects, this just got missed somehow